schematic/utils/general.py

```diff
@@ -69,4 +69,31 @@ def convert_gb_to_bytes(gb: int):
     return: total number of bytes
     """
     bytes_to_return = gb * 1024 * 1024 * 1024
-    return bytes_to_return
\ No newline at end of file
+    return bytes_to_return
+
+
+def entity_type_checking(syn, manifest_id):
+    """
+    check the entity type of the id that needs to be downloaded
+    Args:
+        manifest_id: id of a manifest
+    Return: 
+        type_entity: type of the manifest being returned
+    """
+    # check the type of entity
+    try: 
+        entity_name = syn.get(manifest_id, downloadFile=False)
```

Comment:
Nit: Is this an `entity_name` or is it an `entity`?  You want to try to name your variables what they actually are.